ansible_collections/arista/avd/roles/eos_designs/defaults/main/main.yml

```diff
@@ -88,6 +88,7 @@ platform_settings:
       non_mlag: 330
     feature_support:
       queue_monitor_length_notify: false
+      interface_storm_control: false
```

Comment:
I would vote for a new platform type to support VEOS-LAB as storm-control is valid for all HW as far as I know

Reply:
Let us keep the [ default ] as is.
The default setting for queue_monitor_length_notify should also be removed with AVD 3.0. We kept it there to avoid breaking existing deployments. Let us file a separate PR for that and document the change.

Reply:
And +1 that we could add a default platform for veos, disabling all these knobs.

Reply:
Solved by #1216